var/spack/repos/builtin/packages/spiner/package.py

```diff
@@ -0,0 +1,74 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Spiner(CMakePackage, CudaPackage):
+    """Spiner:
+    Performance portable routines for generic, tabulated, multi-dimensional data"""
+
+    homepage    = "https://github.com/lanl/spiner"
+    url         = "https://github.com/lanl/spiner/archive/refs/tags/1.4.0.tar.gz"
+    git         = "https://github.com/lanl/spiner.git"
+
+    maintainers = ['rbberger']
+
+    version("main", branch="main")
+    version('1.4.0', sha256='c3801b9eab26feabec33ff8c59e4056f384287f407d23faba010d354766f3ac5')
+    version('1.3.0', sha256='85ec31288870ccf203ff99742cd119b538fec7c394e4191e3794af18de8da782')
+    version('1.2.0', sha256='8f0b64db741a521766ac73da8284264f822880bab4d8837911a240629d2bb349')
+
+    # When overriding/overloading varaints, the last variant is always used, except for
+    # "when" clauses. Therefore, call the whens FIRST then the non-whens.
+    # https://spack.readthedocs.io/en/latest/packaging_guide.html#overriding-variants
+    variant("kokkos", default=False, description="Enable kokkos",)
+    variant("openmp", default=False, description="Enable openmp kokkos backend")
+
+    variant("hdf5", default=False, description="Enable hdf5")
+    variant("mpi", default=False, description="Support parallel hdf5")
+
+    variant("python", default=False, description="Python, Numpy & Matplotlib Support")
+    variant("doc", default=False, description="Sphinx Documentation Support")
+    variant("format", default=False, description="Clang-Format Support")
+
+    depends_on("cmake@3.12:")
+    depends_on("catch2@2.13.4:2.13.6")
+
+    # Currently the raw cuda backend of ports-of-call is not supported.
+    depends_on("ports-of-call portability_strategy=Kokkos", when="+kokkos")
+    depends_on("ports-of-call portability_strategy=None", when="~kokkos")
+    for _flag in list(CudaPackage.cuda_arch_values):
+        depends_on("kokkos@3.2.00 cuda_arch=" + _flag, when="+cuda+kokkos cuda_arch=" + _flag)
```

Comment:
It is exactly `3.2.00` or `3.2.00:`?

Reply:
@junghans After checking, it seems to be a minimal version. Updated. Also asked the for a new release of `ports-of-call` to not have to rely on `ports-of-call@main`. After #29723 is merged, I'll update this PR to depend on `ports-of-call@1.1.0:`.

Reply:
@rbberger Since `main` is automatically "newer than" 1.1.0 in Spack versioning, you can make this change now and it will automatically work once the new version is available.